composer.json

```
@@ -11,6 +11,7 @@
   "require": {
     "php": ">=7.1",
     "stuttter/wp-user-signups": "3.1.0",
+    "roots/wp-password-bcrypt": "^1.0",
```

We don't control this plugin so lets add it as a fixed version number instead:
```suggestion
    "roots/wp-password-bcrypt": "1.0.0",
```